sending-a-simple-message-to-a-service/README.md

```diff
@@ -0,0 +1,325 @@
+# Sending a Simple Message to a Service
+
+## What you'll build
```

Comment:
```suggestion
## What you will build
```
It is a best practice to avoid shortened forms. Also, the tone should be reconsidered as this is not a blog and is formal technical documentation. `What you will build` sounds too informal. Shall we say `Overview`?

Reply:
IMO, this is not a formal thing like a spec, but more a tutorial that users would follow along.

Reply:
Changed to Overview anyway.